src/components/View/Lottery/components/DrawInfo/Content.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
Is there a reason for importing `React`? 👀

Reply:
Sorry i forgot to delete it

Reply:
no problemo 💯